### PR TITLE
Switch site theme to pydata sphinx theme

### PR DESCRIPTION
**Pull request by maximlt**

### CODE REVIEW

File: examples/index.ipynb [hunk 1]
```diff
@@ -6,12 +6,12 @@
...
     "## Collection of perceptually accurate colormaps\n",
     "\n",
-    "[Colorcet](https://github.com/pyviz/colorcet) is a collection of \n",
+    "[Colorcet](https://github.com/holoviz/colorcet) is a collection of \n",
     "perceptually acccurate 256-color colormaps for use with Python plotting programs like\n",
-    "[Bokeh](http://bokeh.pydata.org),\n",
-    "[Matplotlib](http://matplotlib.org),\n",
-    "[HoloViews](http://holoviews.org), and\n",
-    "[Datashader](https://github.com/pyviz/datashader). \n",
+    "[Bokeh](https://docs.bokeh.org/en/latest),\n",
```
Comment (jbednar) [inline examples/index.ipynb:11]:
```suggestion
    "[Bokeh](https://docs.bokeh.org),\n",
```

File: examples/user_guide/Continuous.ipynb [hunk 2]
```diff
@@ -6,10 +6,10 @@
...
     "\n",
-    "We will plot them using [Matplotlib](http://matplotlib.org) via [Holoviews](http://holoviews.org), but identical [Bokeh](http://bokeh.pydata.org) palettes are also provided, and both Bokeh palettes and Matplotlib colormaps are usable in [Datashader](https://github.com/pyviz/datashader).  Thus these colormaps can be used in any of those Python packages, as well as any other package that accepts a Python list of normalized RGB tuples or hex colors.  See \n",
-    "[Peter's site](http://peterkovesi.com/projects/colourmaps/) to download versions for other non-Python packages."
+    "We will plot them using [Matplotlib](https://matplotlib.org) via [Holoviews](https://holoviews.org), but identical [Bokeh](https://docs.bokeh.org/en/latest) palettes are also provided, and both Bokeh palettes and Matplotlib colormaps are usable in [Datashader](https://datashader.org).  Thus these colormaps can be used in any of those Python packages, as well as any other package that accepts a Python list of normalized RGB tuples or hex colors.  See \n",
```
Comment (jbednar) [inline examples/user_guide/Continuous.ipynb:11]:
```suggestion
    "We will plot them using [Matplotlib](https://matplotlib.org) via [Holoviews](https://holoviews.org), but identical [Bokeh](https://docs.bokeh.org) palettes are also provided, and both Bokeh palettes and Matplotlib colormaps are usable in [Datashader](https://datashader.org).  Thus these colormaps can be used in any of those Python packages, as well as any other package that accepts a Python list of normalized RGB tuples or hex colors.  See \n",
```

File: doc/conf.py [hunk 3]
```diff
@@ -8,44 +8,49 @@
...
-    'DESCRIPTION': description,
-    'AUTHOR': authors,
-    'WEBSITE_SERVER': 'http://{}.holoviz.org'.format(project),
-    'VERSION': version,
-    'GOOGLE_ANALYTICS_UA': 'UA-154795830-4',
-    'NAV': _NAV ,
-    # by default, footer links are same as those in header
-    'LINKS': _NAV,
-    'SOCIAL': _SOCIAL
+    "last_release": f"v{'.'.join(colorcet.__version__.split('.')[:3])}",
```
Comment (jbednar) [inline doc/conf.py:49]: Should this use `base_version`, or maybe just `version` as defined above?

**Review by jbednar — APPROVED** [review]
Thanks!!

File: doc/conf.py (hunk 3 above)
Comment (maximlt) [reply doc/conf.py:53]: I actually wasn't sure what this parameter what used for, just looked it up, it's used by panel and geoviews to set the link on each page to binder, so it's not relevant for colorcet, I'm just going to remove this entry.

**Pull request merged**